splunk/pytest-splunk-addon
# Acd 4166 Resolve the CIM implementation issues found during testing

Required changes in CIM 
* Search failure due to {{| eval}} command in the Constrain.
* If tags.conf / transforms.conf does not exist, there should not be an error in collection
* Update the Field negative values. Include null, (null)
* Add a log message if an eventtype is not mapped with a data model
* The test case should not pass if no events found for a required fields 
* The test should mention the exact reason with which it was failed. 
* A few fields missing from Authentication Json file
* Add-on is parsed 7 times for the test execution 
* Negative test scenarios not being generated 
* tstats query fails as the Splunk Audit logs are included in the results 
* Data model json for web has invalid conditions 
* Data model schema does not validate data model names.
* Add one liner failure message in the test cases for the report
* test cases for dest_nt_domain & src_nt_domain should not be generated
* Change Data model json file has wrong list of tags
